### PR TITLE
impl Future for Rust

### PR DESCRIPTION
**Pull request by aturon**
This is a draft blog post for launching the impl period.

**WG leads: I need a one-liner for each group**. I've written about half of them. Please take a look, and leave a comment to provide (or change) your one-liner ASAP.

cc @rust-lang/core

### CODE REVIEW

**Comment by fitzgen** [comment]
**bindgen:** Make FFI'ing to C and C++ easy, automatic, and robust!

**Comment by steveklabnik** [comment]
`WG-docs-rbe`: Teaching others Rust in the browser

**Comment by carols10cents** [comment]
WG-infra-crates.io: Pay down some technical debt and implement long-desired features on a production Rust web application!

**Comment by QuietMisdreavus** [comment]
`WG-docs-rustdoc`: Help make docs beautiful for everyone!

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+<table>
+<tr>
+    <td><b>WG-infra-crates.io</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-perf</b></td>
+    <td></td>
```
Comment (Mark-Simulacrum) [inline _posts/2017-09-18-impl-future-for-rust.md]: Let's make sure Rust gets faster.

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+</tr>
+<tr>
+    <td><b>WG-infra-host</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-rustbuild</b></td>
+    <td></td>
```
Comment (Mark-Simulacrum) [inline _posts/2017-09-18-impl-future-for-rust.md]: Ensuring stability and speed in the compiler build.

I'm not super happy with this one...

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+</tr>
+<tr>
+    <td><b>WG-infra-crater2</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-secure</b></td>
+    <td></td>
```
Comment (erickt) [inline _posts/2017-09-18-impl-future-for-rust.md]: Securing Rust's infrastructure to make it follow best security practices, consistent, and easy to update.

**Comment by BurntSushi** [comment]
`WG-libs-simd`: Implement SIMD vendor intrinsics!

**Comment by petrochenkov** [comment]
@BurntSushi 
"Help Rust to directly access hardware's advanced data processing capabilities!" :)

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+## A few points of order
+
+There are a few online venues for keeping in the loop with working group activity:
+
+- There is a [dedicated Gitter community](https://gitter.im/rust-impl-period/)
+  with channels for each working group, as well as
+  a [global channel](https://gitter.im/rust-impl-period/Lobby) for talking about
+  the process as a whole, or getting help finding your way to a working group.
+
+- The brand-new [findwork](https://www.rustaceans.org/findwork) site, which
```
Comment (nrc) [inline _posts/2017-09-18-impl-future-for-rust.md]: Could use https://www.rustaceans.org/findwork/impl to go straight to the impl period issues

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+
+<table>
+<tr>
+    <td><b>WG-dev-tools-rls</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-vscode</b></td>
```
Comment (nrc) [inline _posts/2017-09-18-impl-future-for-rust.md]: Can we add WG-dev-tools-clients please? "Implement new clients for the RLS - Atom, Sublime Text, Visual Studio, etc."

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+    <td></td>
```
Comment (nrc) [inline _posts/2017-09-18-impl-future-for-rust.md]: Improve Rust's IDE experience for Visual Studio Code users

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Dev tools team
+
+<table>
+<tr>
+    <td><b>WG-dev-tools-rls</b></td>
+    <td></td>
```
Comment (nrc) [inline _posts/2017-09-18-impl-future-for-rust.md]: Help make Rust's IDE experience first class

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+</tr>
+<tr>
+    <td><b>WG-dev-tools-vscode</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-rustfmt</b></td>
+    <td></td>
```
Comment (nrc) [inline _posts/2017-09-18-impl-future-for-rust.md]: Help make Rust's code the prettiest!

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 9]
```diff
@@ -0,0 +1,312 @@
...
+</tr>
+<tr>
+    <td><b>WG-infra-perf</b></td>
+    <td>Let's make sure Rust gets faster.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-crater</b></td>
+    <td></td>
```
Comment (aidanhs) [inline _posts/2017-09-18-impl-future-for-rust.md]: Testing the compiler against the Rust ecosystem.

File: _posts/2017-09-18-impl-future-for-rust.md [hunk 10]
```diff
@@ -0,0 +1,312 @@
...
+</tr>
+<tr>
+    <td><b>WG-infra-secure</b></td>
+    <td>Help us implement best practices for Rust's infrastructure!</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-infra-host</b></td>
+    <td></td>
```
Comment (aidanhs) [inline _posts/2017-09-18-impl-future-for-rust.md]: Managing the services that keep the Rust machine running.

**Pull request merged**